### DISCUSSION TITLE
Is there a way to enable ruler?

### DISCUSSION THREAD

**Discussion by hexdigest** [NvChad/NvChad#2157] [General]
This is something I miss from my previous configuration

```lua
opt.ruler = true
```

in my init.lua doesn't do anything, cursor position isn't being displayed anywhere in the status line

**Comment by siduck** [comment]
@hexdigest add that in your custom init.lua

**Reply by hexdigest** [reply]
Hey @siduck 

This is what I already did but the ruler doesn't show up anywhere in nvchad status line

**Reply by siduck** [reply]
@hexdigest thats because you didnt include that in our statusline modules, so only default modules are showed and ruler isnt present in the default modules, try to override statusline modules or try  minimal/vscode/vscode_colored in your statusline theme option.

**Reply by hexdigest** [reply]
Thank you, that helped:

```lua
M.ui = {
  theme = 'solarized_dark',
  statusline = {
    theme = "vscode_colored",
  },
}
```
